metadata-service/openapi-servlet/src/main/java/io/datahubproject/openapi/config/SpringWebConfig.java

```diff
@@ -38,24 +44,25 @@ public class SpringWebConfig implements WebMvcConfigurer {
   private static final Set<String> OPENLINEAGE_PACKAGES =
       Set.of("io.datahubproject.openapi.openlineage");
 
-  public static final Set<String> NONDEFAULT_OPENAPI_PACKAGES;
-
-  static {
-    NONDEFAULT_OPENAPI_PACKAGES = new HashSet<>();
-    NONDEFAULT_OPENAPI_PACKAGES.addAll(OPERATIONS_PACKAGES);
-    NONDEFAULT_OPENAPI_PACKAGES.addAll(SCHEMA_REGISTRY_PACKAGES);
-    NONDEFAULT_OPENAPI_PACKAGES.addAll(OPENLINEAGE_PACKAGES);
-    NONDEFAULT_OPENAPI_PACKAGES.addAll(V1_PACKAGES);
-    NONDEFAULT_OPENAPI_PACKAGES.addAll(V2_PACKAGES);
-    NONDEFAULT_OPENAPI_PACKAGES.addAll(V3_PACKAGES);
-  }
```

Comment:
the new default openapi page will now have all of the above packages right? AKA we don't want to hide them anymore

Reply:
The difference is there is actually no longer a catch-all group. If new endpoints are added they must either fall into an existing group or a new group can be added. This is more explicit and avoids wiring up endpoints inadvertently with some misplaced annotations.